packages/salesforcedx-utils-vscode/src/telemetry/reporters/appInsights.ts

```diff
@@ -221,6 +186,6 @@ export class TelemetryReporter extends Disposable {
         resolve(void 0);
       }
     });
-    return Promise.all([flushEventsToAI, flushEventsToLogger]);
+    return Promise.all([flushEventsToAI]);
```

Comment:
```suggestion
    return flushEventsToAI;
```

can reduce the method to return inline